gmso/core/element.py

```diff
@@ -91,9 +97,9 @@ def element_by_name(name):
         Return an element from the periodic table if the name is found,
         otherwise return None
     """
-    name_trimmed = sub(r'[0-9 -]', '', name).lower()
-    msg = '''Numbers and spaces are not considered when searching by element name. \n
-             {} became {}'.format(name, name_trimmed)'''
+    name_trimmed = sub(r"[0-9 -]", "", name).lower()
+    msg = """Numbers and spaces are not considered when searching by element name. \n
+             {} became {}'.format(name, name_trimmed)"""
```

Comment:
Is this lone `'` on line 102 meant to be there?

Reply:
Fixed to use format strings from Python.